bpf/lib/ipv4.h

```diff
@@ -111,7 +116,11 @@ ipv4_frag_register_datagram(struct __ctx_buff *ctx, int l4_off,
 	if (ret < 0)
 		return ret;
 
-	map_update_elem(&IPV4_FRAG_DATAGRAMS_MAP, frag_id, ports, BPF_ANY);
+	if (map_update_elem(&IPV4_FRAG_DATAGRAMS_MAP, frag_id, ports, BPF_ANY)
+	    < 0)
```

Comment:
```suggestion
	if (map_update_elem(&IPV4_FRAG_DATAGRAMS_MAP, frag_id, ports, BPF_ANY) < 0)
```
More readable on one line? Or break inside `map_update_elem()` otherwise?

Reply:
I assumed `checkpatch` would complain on lines with > 80 chars but the limit is actually set to 100 so I think  we are good to go with the one line option :+1:

Reply:
Right, it's still recommended to keep lines “short”, but the limit for checkpatch complaints was raised in the kernel repo a few months ago. In that case I believe it does read better on one line.

Reply:
Also for your information, we have a few false positives with checkpatch at the moment, that I intend to address. There's a list on https://github.com/cilium/cilium/issues/12467.

Reply:
Re-reading that line -- You could as well remove the `< 0`, since the helper always return `0` on success.